lib/reverse_sentence.rb

```diff
@@ -1,6 +1,19 @@
+
 # A method to reverse the words in a sentence, in place.
 # Time complexity: ?
 # Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+
+def reverse_sentence(test_string)
```

Comment:
Right now you're only reversing the sentence.  You then need to do another loop reversing each word.  Something to work on.